group-coordinator/src/main/java/org/apache/kafka/coordinator/group/OffsetMetadataManager.java

```diff
@@ -620,8 +621,9 @@ public boolean cleanupExpiredOffsets(String groupId, List<Record> records) {
         });
 
         if (!expiredPartitions.isEmpty()) {
-            log.info("[GroupId {}] Expiring offsets of partitions (allOffsetsExpired={}): {}",
-                groupId, allOffsetsExpired, String.join(", ", expiredPartitions));
+            log.info("[GroupId {}] Expiring {} offsets (allOffsetsExpired={}) in {} milliseconds.",
```

Comment:
I don't really get the value of adding the time here. What's the reasoning behind it? However, I think that it would make sense to add a log in `cleanupGroupMetadata` which contains the time taken and the number of records generated.